### PR TITLE
feat(entities): Implement Rest of Chicken Methods

### PR DESCRIPTION
**Pull request by thelooter**
# Description

# Checklist
The following items should be checked before the pull request can be merged.
- [ ] Code follows existing style.
- [ ] Unit tests added (if applicable).

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/entity/ChickenMock.java [hunk 1]
```diff
@@ -46,30 +51,25 @@ public boolean isBreedItem(@NotNull ItemStack stack)
...
+		return this.isChickenJockey;
 	}
 
 	@Override
 	public void setIsChickenJockey(boolean isChickenJockey)
 	{
-		//TODO: Auto-generated method stub
-		throw new UnimplementedOperationException();
-
+		this.isChickenJockey = isChickenJockey;
```
Comment (Insprill) [inline src/main/java/be/seeseemelk/mockbukkit/entity/ChickenMock.java:60]: How does this work in Paper? Should we be spawning the baby zombie and setting it to ride the chicken?

**Review by Insprill — CHANGES_REQUESTED** [review]

File: src/main/java/be/seeseemelk/mockbukkit/entity/ChickenMock.java (hunk 1 above)
Comment (thelooter) [reply src/main/java/be/seeseemelk/mockbukkit/entity/ChickenMock.java:60]: From what I can tell it's just a variable thats set. I don't see the spawning of the baby zombie anywhere

Comment (Thorinwasher) [reply src/main/java/be/seeseemelk/mockbukkit/entity/ChickenMock.java:60]: Yep, it's just a boolean that is set to indicate things for the chicken, for example not to lay eggs, the xp rewards, and whether to despawn when player is too far away.

**Review by Insprill — APPROVED** [review]
LGTM

**Pull request merged**